ui-particles-angular/projects/ui-policy-studio-angular/src/lib/components/flow-form-dialog/flow-proxy-form-dialog/gio-ps-flow-proxy-form-dialog.component.html

```diff
@@ -0,0 +1,80 @@
+<!--
+
+    Copyright (C) 2015 The Gravitee team (http://gravitee.io)
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+            http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+-->
+<h2 mat-dialog-title class="title">
+  {{ mode === 'create' ? 'Create a new flow' : 'Edit flow' }}
+
+  <button class="title__closeBtn" mat-icon-button aria-label="Close dialog" [mat-dialog-close]="false">
+    <mat-icon svgIcon="gio:cancel"></mat-icon>
+  </button>
+</h2>
+
+<mat-dialog-content *ngIf="flowFormGroup" class="content" [formGroup]="flowFormGroup">
+  <p>
+    Flows allow you to apply different policies per HTTP path and/or method. For example you can reroute calls based on the URL specified.
+  </p>
+
+  <div class="content__row">
+    <mat-form-field>
+      <mat-label>Flow name</mat-label>
+      <input matInput formControlName="name" cdkFocusInitial />
+      <mat-hint>Names will be automatically generated with path and method if left blank</mat-hint>
+    </mat-form-field>
+  </div>
+  <div class="content__row">
+    <mat-form-field>
+      <mat-label>Operator</mat-label>
+      <mat-select formControlName="pathOperator">
+        <mat-option value="EQUALS">Equals</mat-option>
+        <mat-option value="STARTS_WITH">Starts with</mat-option>
+      </mat-select>
+      <mat-hint>Path operator</mat-hint>
+    </mat-form-field>
+
+    <mat-form-field>
+      <mat-label>Path</mat-label>
+      <input matInput formControlName="path" />
+      <mat-hint>Publish or Subscribe channel. This is the path of your request.</mat-hint>
```

Comment:
issue: it is not the right hint for an HTTP Proxy
suggestion: use `Path` as described in figma
<img width="474" alt="image" src="https://github.com/gravitee-io/gravitee-ui-particles/assets/13161768/4da1a8f3-d8da-4f43-8730-9a1a098c0742">